### PR TITLE
Create sensitive_data_detection.md

### PR DESCRIPTION
**Pull request by pgkamat**
### What does this PR do?

New documentation for Sensitive data detection

### Motivation

New product

### Preview

https://docs-staging.datadoghq.com/pranay/new-sds-docs/logs/log_configuration/sensitive_data_detection/

### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/logs/log_configuration/sensitive_data_detection.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+---
+title: Sensitive Data Detection
+kind: documentation
+aliases:
+    - /sensitive_data_detection/
```
Comment (sarina-dd) [inline content/en/logs/log_configuration/sensitive_data_detection.md:5]: Did this page previously exist? If it's brand new, we can remove the alias as this is used solely for redirect.

File: content/en/logs/log_configuration/sensitive_data_detection.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+further_reading:
```
Comment (sarina-dd) [inline content/en/logs/log_configuration/sensitive_data_detection.md:6]:
```suggestion
beta: true
further_reading:
```

File: content/en/logs/log_configuration/sensitive_data_detection.md (hunk 1 above)
Comment (pgkamat) [reply content/en/logs/log_configuration/sensitive_data_detection.md:5]: This is a new page. So I will remove the alias.

File: content/en/logs/log_configuration/sensitive_data_detection.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+Often businesses are required to identify, remediate, and prevent the exposure of such sensitive data within their logs due to organizational policies, compliance requirements, industry regulations, and privacy concerns. This is especially true for industries such as banking, financial services, healthcare, and insurance among others.
+
+## Sensitive data scanner
+
+Sensitive data scanner is a stream-based, pattern matching service that you can use to identify, tag, and optionally redact or hash sensitive data. With implementation, your security and compliance teams can introduce a new line of defense in preventing sensitive data from leaking outside your organization.
+
+Datadog's Scanner Library offers a wide variety of rules for commonly detected patterns such as email addresses, SSN, credit card numbers, API keys, authorization tokens, and others.
+
+{{< img src="dashboards/high-density-mode.png" alt="The high-density mode display"  style="width:90%;">}}
```
Comment (sarina-dd) [inline content/en/logs/log_configuration/sensitive_data_detection.md:28]: @pgkamat Hi there! Completed an editorial review and the content LGTM. Let me know when the new images are in and I can do a final review!

File: content/en/logs/log_configuration/sensitive_data_detection.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+further_reading:
+- link: "/security/logs/"
+  tag: "Documentation"
+  text: "Security"
+- link: "/logs/explorer/"
+  tag: "Documentation"
+  text: "Log Explorer"
+---
+
+<div class="alert alert-info">This page is about Sensitive Data Scanner, which is in currentl in private beta. Contact <a href="https://www.datadoghq.com/support/">Datadog Support</a> if you want to learn more or participate in the beta program. </div>
```
Comment (yoannrobin) [inline content/en/logs/log_configuration/sensitive_data_detection.md:14]:
```suggestion
<div class="alert alert-info">This page is about Sensitive Data Scanner, which is currently in private beta. Contact <a href="https://www.datadoghq.com/support/">Datadog Support</a> if you want to learn more or participate in the beta program. </div>
```

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**